policyengine_us/variables/household/income/spm_unit/spm_unit_oecd_equivalised_net_income.py

```diff
@@ -14,5 +14,5 @@ class spm_unit_oecd_equiv_net_income(Variable):
 
     def formula(spm_unit, period, parameters):
         number_of_people = spm_unit.nb_persons()
```

Comment:
then use this instead of the members to simplify the unit test
```suggestion
        number_of_people = spm_unit("spm_unit_size", period)
```